app/src/main/java/app/revanced/manager/util/PM.kt

```diff
@@ -115,6 +115,14 @@ class PM(
 
     fun PackageInfo.label() = this.applicationInfo.loadLabel(app.packageManager).toString()
 
+    fun getVersionCode(packageInfo: PackageInfo): Int {
+        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            packageInfo.longVersionCode.toInt()
+        } else {
+            packageInfo.versionCode
+        }
+    }
```

Comment:
There is an official function that can be used instead: https://developer.android.com/reference/kotlin/androidx/core/content/pm/PackageInfoCompat#getLongVersionCode(android.content.pm.PackageInfo)

Reply:
`getLongVersionCode` was added in API 28, we target a minimum of API 26.